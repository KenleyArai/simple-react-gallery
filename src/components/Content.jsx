import styled from "styled-components";
import PropTypes from "prop-types";

import theme from "../theme";

const Content = styled.div`
  background: ${props => props.theme.secondary};
`;

Content.propTypes = {
  theme: PropTypes.shape({
    primary: PropTypes.string,
    secondary: PropTypes.string,
    tertiary: PropTypes.string
  })
};

Content.defaultProps = {
  theme: { ...theme }
};

export default Content;
