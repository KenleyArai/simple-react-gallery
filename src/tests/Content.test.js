import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
import "jest-styled-components";
configure({ adapter: new Adapter() });

import Content from "components/Content";

describe("Content", () => {
  it("should render without Content", () => {
    const wrapper = shallow(<Content />);
    expect(wrapper).toMatchSnapshot();
  });
});
