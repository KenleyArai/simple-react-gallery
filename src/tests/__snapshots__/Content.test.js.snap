// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render without Content 1`] = `
<div
  className=""
/>
`;
